File: cropme/src/main/java/com/takusemba/cropme/OnCropListener.java
package com.takusemba.cropme;

import android.graphics.Bitmap;

/**
 * Listener to return an cropped image.
 **/
public interface OnCropListener {

    /**
     * called when cropping is successful
     *
     * @param bitmap result bitmap
     **/
    void onSuccess(Bitmap bitmap);

    /**
     * called when cropping is failed
     **/
    void onFailure();
}

File: cropme/src/main/java/com/takusemba/cropme/OnCropListener.kt
package com.takusemba.cropme

import android.graphics.Bitmap

/**
 * Listener to return an cropped image.
 */
interface OnCropListener {

  /**
   * called when cropping is successful
   *
   * @param bitmap result bitmap
   */
  fun onSuccess(bitmap: Bitmap)

  /**
   * called when cropping is failed
   */
  fun onFailure(e: Exception)
}
